core/modules/modPrune.class.php

```diff
@@ -227,6 +227,6 @@ public function remove($options = '')
 	{
 		$sql = [];
 
-		return $this->_remove($sql, $options);
+		return $this->_remove($sql,$options);
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma
**Severity**: ERROR
**File**: core/modules/modPrune.class.php L230

No space found after comma in argument list

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-610360255>*